cmd/zoekt-sourcegraph-indexserver/index.go

```diff
@@ -67,6 +68,9 @@ type IndexOptions struct {
 	// The number of threads to use for indexing shards. Defaults to the number of available
 	// CPUs. If the server flag -cpu_fraction is set, then this value overrides it.
 	ShardConcurrency int32
+
+	// The Sourcegraph tenant
+	TenantId int
```

Comment:
Nit: TenantID feels more Go-ish to me

Reply:
https://google.github.io/styleguide/go/decisions#initialisms Google's style guide says so too